apps/system/test/marionette/lib/video_player.js

```diff
@@ -58,13 +58,16 @@ VideoPlayer.prototype = {
   },
 
   invokeControls: function() {
-    if (!this.visibleControls()) {
-      this.rootElement.tap();
-
+    if(this.visibleControls()) {
+      this.rootElement.tap(0, 0);
```

Comment:
Yeah again the problem here is `visibleControls()` doesn't tell us if the controls are about to disappear, but the problem is this tap can be a toggle and if controls are visible, then disappear, and then we tap, we can get in a state where we just activated the controls. But this is a minefield, so I would say we do whatever makes the test the most robust.